src/lib/io/include/sol-fs-storage.h

```diff
@@ -106,6 +106,15 @@ int sol_fs_read_raw(const char *name, struct sol_buffer *buffer);
         return -EINVAL; \
     }
 
+/**
+ * @brief Read an uint8_t in storage
+ *
+ * @param name name of property. It will look for a file on filesystem with
+ * this name
+ * @param value value sets the buffer that will be read
```

Comment:
What do you think about "@param value value helps to set a struct @ref sol_buffer"? @cabelitos